ingestion/src/metadata/profiler/metrics/hybrid/histogram.py

```diff
@@ -110,18 +111,23 @@ def _get_bins(
             res_min (float): minimum value
             res_max (float): maximum value
         """
-        # freedman-diaconis rule
-        bin_width = self._get_bin_width(float(res_iqr), res_row_count)  # type: ignore
-        num_bins = math.ceil((res_max - res_min) / bin_width)  # type: ignore
+        # preinint num_bins over 100.  On the normal path freedman-diaconis will readjust according to the algorithm
+        # when we must fallback to sturges rule due to res_iqr being None, then num_bins will be readjusted.
+        max_bin_count = 100
+        num_bins = max_bin_count + 1
+        if res_iqr:
```

Comment:
@ksirmons we'll need to do `if res_iqr is not None`, otherwise using `if res_iqr` will yield `False` if res_iqr=0.0`

Reply:
Good catch.